final-project/static/scripts.js

```diff
@@ -61,16 +63,23 @@ function retrieveJSON(product_id) {
 
   // Retrieve the desired product info through a promise.
   getProductJSON.then(function(response) {
-    console.log("retrieveJSON returned response")
+
+    console.log(" ")
+    console.log("Success: retrieveJSON returned response")
```

Comment:
You can use `\n` in your String to signify new lines.

Reply:
Thanks! This was bugging me, but I did not think to use \n